pandas/core/indexes/multi.py

```diff
@@ -1010,14 +1010,14 @@ def _to_safe_for_reshape(self):
 
     def to_frame(self, index=True):
         """
-        Create a DataFrame with the columns the levels of the MultiIndex
+        Create a DataFrame with the levels of the MultiIndex as columns.
 
         .. versionadded:: 0.20.0
 
         Parameters
         ----------
         index : boolean, default True
-            return this MultiIndex as the index
+            Set the index of the returned DataFrame as self.
```

Comment:
I personally would not use "self" in docstrings. For users that are not very familiar with writing classes themselves (and you don't need to do this for using pandas) this will sound very strange.

Reply:
Fair enough.  Fixed.